### PR TITLE
Hw20

### PR DESCRIPTION
**Pull request by VyacheslavShrot**

### CODE REVIEW

File: unittests/tests_functions.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+class TestFibonacci(unittest.TestCase):
+
+    def setUp(self) -> None:
+        self.function = Fibonacci()
+
+    def test_1(self):
+        self.assertEqual(self.function(1), 1)
+
+    def test_float(self):
```
Comment (mikolaz27) [inline unittests/tests_functions.py:14]: Use more descriptive test case names to help others (and your future self) understand what is being tested. Instead of test_1, use something like test_fibonacci_of_1_returns_1.

File: unittests/tests_functions.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    def test_str(self):
+        self.assertRaises(ValueError, self.function, "0")
+
+    def test_negative(self):
+        self.assertRaises(ValueError, self.function, -1)
+
+
+class TestFormattedName(unittest.TestCase):
+
+    def test_lower_name(self):
```
Comment (mikolaz27) [inline unittests/tests_functions.py:36]: Change the test methods' names to be more descriptive of what they are testing.

File: unittests/tests_functions.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    def test_upper_name(self):
+        self.assertEqual(formatted_name('ARTUR', 'VINI', middle_name='A'), 'Artur A Vini')
+
+    def test_full_name(self):
+        self.assertEqual(formatted_name('Artur', 'Vini', middle_name='A'), 'Artur A Vini')
+
+    def test_without_middle_name(self):
+        self.assertEqual(formatted_name('Artur', 'Vini'), 'Artur Vini')
+
+    def test_int(self):
```
Comment (mikolaz27) [inline unittests/tests_functions.py:48]: You may also want to consider adding more test cases, such as testing for edge cases, like names with special characters, empty strings, or long names.
